hw/ip/csrng/data/csrng.hjson

```diff
@@ -7,6 +7,14 @@
     { protocol: "tlul", direction: "device" }
   ],
   param_list: [
+    // Random netlist constants
+    { name:      "RndCnstCsKeymgrDivProduction",
+      desc:      "Compile-time random bits for cs state group diversification value",
+      type:      "cs_ctrl_pkg::cs_keymgr_div_t",
```

Comment:
Shouldn't this be cs**rng**_ctrl_pkg?

Also, I do not see the type definitions in this PR... did you forget to add the file?

Reply:
ah...something tells me there is no cross checking between hjson and verilog files. But you are probably on to something here, will revisit.